### PR TITLE
Test DDL compiler with output from SQL parser

### PR DESCRIPTION
**Pull request by bkerley**
Also includes: support code (read: Makefile) to make buildbot work

Fixes RTS-230

### CODE REVIEW

File: src/riak_kv_ddl_compiler.erl [hunk 1]
```diff
@@ -1423,36 +1432,71 @@ simple_valid_map_extract_test() ->
...
+                                   position = 1,
+                                   type     = Map1}
+                   ]),
     {module, Module} = make_helper_mod(DDL),
     Obj = {{2, {3}, {4}}},
     Path = ["Top_Map", "Level_1_map1", "in_Map_1"],
     Res = (catch Module:extract(Obj, Path)),
     ?assertEqual(3, Res).
 
+complex_ddl_test() ->
```
Comment (gordonguthrie) [inline src/riak_kv_ddl_compiler.erl]: Should this test not be in the ddl_compiler?

**Comment by gordonguthrie** [comment]
+1 38f41531ae3cc86abb4416c830b75d4bc1f5f40f

**Comment by gordonguthrie** [comment]
+1 38f41531ae3cc86abb4416c830b75d4bc1f5f40f

**Comment by bkerley** [comment]
@borshop merge

**Pull request merged**